### PR TITLE
loader: Fix Clang warnings.

### PR DESCRIPTION
**Pull request by null77**
error: equality comparison with extraneous parentheses

error: comparison of array 'layer_prop->functions.str_gipa' equal to a null pointer is always false

PTAL.. some warnings I turned up when building the loader with Clang (on Windows).

### CODE REVIEW

File: loader/vk_loader_platform.h [hunk 1]
```diff
@@ -234,7 +234,6 @@ loader_platform_thread_cond_broadcast(loader_platform_thread_cond *pCond) {
 #ifdef __cplusplus
 #include <iostream>
 #include <string>
-using namespace std;
```
Comment (MarkY-LunarG) [inline loader/vk_loader_platform.h]: Why are you removing this?  I noticed some of the layers that include this don't re-define "using namespace std;".  Are you only seeing this warning in a certain layer(s)?

Comment (mark-lunarg) [inline loader/vk_loader_platform.h]: @MarkY-LunarG, it is standard in LVL to avoid 'using' directives and to explicitly supply namespaces.

**Comment by mikew-lunarg** [comment]
If used at all, "using namespace" should only be in .cpp files, not headers.

On Thu, Dec 15, 2016 at 11:32 AM, Mark Lobodzinski <notifications@github.com

File: loader/vk_loader_platform.h (hunk 1 above)
Comment (null77) [inline loader/vk_loader_platform.h]: @MarkY-LunarG confirming what Mike and Mark said, it is a compiler warning (Clang in this case) to import namespaces in headers. I was seeing this when building layers, but @mark-lunarg asked me to move it into a loader-specific CL.

**Pull request merged**